charts/stfc-cloud-logging/values.yaml

```diff
@@ -0,0 +1,90 @@
+fluent-operator:
+  fluentbit:
+    enable: true
+    resources:
+      limits:
+        cpu: 500m
+        memory: 400Mi
+      requests:
+        cpu: 10m
+        memory: 25Mi
+    annotations:
+      fluentbit.io/exclude: "true"
+      promtheus.io/scrape: "true"
+
+    input:
+      tail:
+        enable: true
+      systemd:
+        enable: true
+
+    output:
+      # To enable opensearch, comment out the below line, and uncomment the block
+      opensearch: {}
+      # For a complete set of opensearch options see
+      # https://github.com/fluent/helm-charts/blob/62a6d1c8ea98f40141c479d24ce3fe83a7156b66/charts/fluent-operator/charts/fluent-bit-crds/crds/fluentbit.fluent.io_outputs.yaml#L2532
+      # opensearch:
+      #   host: <opensearch_host>
+      #   port: <opensearch_port>
+      #   # Credentials should be set as a secret, see secrets.yaml.template
+      #   httpUser:
+      #     valueFrom:
+      #       secretKeyRef:
+      #         name: opensearch-credentials
+      #         key: username
+      #   httpPassword:
+      #     valueFrom:
+      #       secretKeyRef:
+      #         name: opensearch-credentials
+      #         key: password
+      #   logstashFormat: On
+      #   logstashPrefix: "<some_index_prefix>"
+      #   # Alternatively, logstash can be disabled, and a static index name specified
+      #   # index: "<some_index_name>"
+      #   replaceDots: true
+      #   type: flb_type
+      #   traceError: true
+      #   suppressTypeName: On
+      #   timeKey: "@timestamp"
+      #   tls:
+      #     verify: false # If your opensearch instance supports TLS, this can be set to true
+      #     debug: 1 # Level 1 is Error only
+
+      # Disable loki if loki-stack is not in use
+      loki:
+        enable: true
+        host: loki-stack
+        port: 3100
+        tenantID: ""
+        labels: ["job=fluent-bit"]
+        removeKeys: []
+        autoKubernetesLabels: "on"
+
+      prometheusMetricsExporter:
+        metricsExporter:
+          host: "0.0.0.0"
+          port: 2020
+          addLabels:
+            app: "fluentbit"
+      
+      # Additional outputs (elasticsearch/kafka...) can be added below, see:
+      # https://github.com/fluent/fluent-operator/tree/master/charts/fluent-operator/templates
+
+    filter:
+      kubernetes:
+        enable: true
+        labels: true
+        annotations: false
+      containerd:
+        enable: true
+      systemd:
+        enable: true
+    
+    serviceMonitor:
+      enable: true
+      interval: 30s
+      path: /api/v2/metrics/prometheus
+      scrapeTimeout: 10s
+      port: http-metrics
```

Comment:
If any of these values are default in the upstream chart (including any other values previously mentioned) we should drop them and just inherit it from the upstream chart

This makes it dead clear what we're changing vs leaving as default